src/Components/CategoryCountContainerPieVR.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/prop-types */
+import { Cell, Pie, PieChart, Tooltip } from "recharts";
+import { useSelector } from "react-redux";
+import { LIGHT_MODE } from "../redux/light-dark/lightDarkTypes";
+import { COLORS } from "../utils/data";
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+const CategoryCountContainerPieVR = ({ arr }) => {
+  const mode = useSelector((store) => store.lightDARK_MODE.lightDARK_MODE);
```

Comment:
@9xVibee the casing seems off, these are reducer and redux state right? this can be in camel casing